contrib/go-redis/redis.v8/option.go

```diff
@@ -58,3 +59,11 @@ func WithAnalyticsRate(rate float64) ClientOption {
 		}
 	}
 }
+
+// WithExcludeRawCommand sets the option to exclude the raw command
+// for when the command/cache might contain sensitive data
```

Comment:
I don't understand this part. The Datadog Agent obfuscates this tag and hides all parameters. Why would the fact that it contains sensitive data be an issue?
```suggestion
// SkipRawCommand determines the integration to skip setting the raw command
// in the redis.raw_command tag of a span.
```
As for the doc, we should be a bit more specific on what it does.